static/app/views/settings/project/tempest/CredentialRow.tsx

```diff
@@ -66,3 +72,35 @@ export function CredentialRow({
     </Fragment>
   );
 }
+
+type StatusTagProps = {
+  type: 'error' | 'success' | 'info';
+  message?: string;
+};
+
+const STATUS_CONFIG = {
+  error: {label: 'Error', type: 'error'},
+  success: {label: 'Active', type: 'default'},
+  info: {label: 'Pending', type: 'info'},
+} as const;
+
+function StatusTag({type, message}: StatusTagProps) {
```

Comment:
We have two different kinds of `type` here which are distinct but can easily be confused with. Maybe find a different name for either?